mifospay/src/main/java/org/mifos/mobilewallet/mifospay/auth/presenter/LoginPresenter.java

```diff
@@ -38,14 +40,20 @@ public LoginPresenter(UseCaseHandler useCaseHandler, PreferencesHelper preferenc
         this.mUsecaseHandler = useCaseHandler;
         this.preferencesHelper = preferencesHelper;
     }
-
+    /**
+     * Attaches View to the Presenter
+     */
     @Override
     public void attachView(BaseView baseView) {
         mLoginView = (AuthContract.LoginView) baseView;
         mLoginView.setPresenter(this);
     }
 
-
+    /**
+     * A function to  authenticate a user based on the credentials
+     * @param username A variable of the type String
```

Comment:
```suggestion
     * @param username Username given by the user.
```
Do same for the password.